frontend/test/jest-setup-env.js

```diff
@@ -1,3 +1,4 @@
+import "@testing-library/jest-dom";
```

Comment:
it's needed for typescript. I removed stub package and typescript started complaining about it even if types were always included into the main package